challenges/08-coding-interview-prep/rosetta-code.json

```diff
@@ -13377,6 +13377,1144 @@
+        },
+        {
+          "text":
+            "'<code>solveApollonius('+JSON.stringify(tests[2][0])+','+JSON.stringify(tests[2][1])+','+JSON.stringify(tests[2][2])+',-1,1,1)</code> should return <code>'+JSON.stringify(results[2])+'</code>.'",
+          "testString":
+            "assert.deepEqual(solveApollonius(tests[2][0],tests[2][1],tests[2][2],-1,1,1),results[2],'<code>solveApollonius('+JSON.stringify(tests[2][0])+','+JSON.stringify(tests[2][1])+','+JSON.stringify(tests[2][2])+',-1,1,1)</code> should return <code>'+JSON.stringify(results[2])+'</code>.');"
+        },
+        {
+          "text":
+            "'<code>solveApollonius('+JSON.stringify(tests[3][0])+','+JSON.stringify(tests[3][1])+','+JSON.stringify(tests[3][2])+',1,-1,-1)</code> should return <code>'+JSON.stringify(results[3])+'</code>.'",
+          "testString":
+            "assert.deepEqual(solveApollonius(tests[3][0],tests[3][1],tests[3][2],1,-1,-1),results[3],'<code>solveApollonius('+JSON.stringify(tests[3][0])+','+JSON.stringify(tests[3][1])+','+JSON.stringify(tests[3][2])+',1,-1,-1)</code> should return <code>'+JSON.stringify(results[3])+'</code>.');"
+        },
+        {
+          "text":
+            "'<code>solveApollonius('+JSON.stringify(tests[4][0])+','+JSON.stringify(tests[4][1])+','+JSON.stringify(tests[4][2])+',1,-1,1)</code> should return <code>'+JSON.stringify(results[4])+'</code>.'",
+          "testString":
+            "assert.deepEqual(solveApollonius(tests[4][0],tests[4][1],tests[4][2],1,-1,1),results[4],'<code>solveApollonius('+JSON.stringify(tests[4][0])+','+JSON.stringify(tests[4][1])+','+JSON.stringify(tests[4][2])+',1,-1,1)</code> should return <code>'+JSON.stringify(results[4])+'</code>.');"
+        },
+        {
+          "text":
+            "'<code>solveApollonius('+JSON.stringify(tests[5][0])+','+JSON.stringify(tests[5][1])+','+JSON.stringify(tests[5][2])+',-1,1,-1)</code> should return <code>'+JSON.stringify(results[5])+'</code>.'",
+          "testString":
+            "assert.deepEqual(solveApollonius(tests[5][0],tests[5][1],tests[5][2],-1,1,-1),results[5],'<code>solveApollonius('+JSON.stringify(tests[5][0])+','+JSON.stringify(tests[5][1])+','+JSON.stringify(tests[5][2])+',-1,1,-1)</code> should return <code>'+JSON.stringify(results[5])+'</code>.');"
+        }
+      ],
+      "id": "5a23c84252665b21eecc7f8d",
+      "challengeType": 5,
+      "releasedOn": "August 22, 2018",
+      "files": {
+        "indexjs": {
+          "key": "indexjs",
+          "ext": "js",
+          "name": "index",
+          "contents": [
+            "function solveApollonius (c1, c2, c3, s1, s2, s3) {",
+            "  // Good luck!",
+            "}"
+          ],
+          "head": [],
+          "tail": [
+            "var tests=[",
```

Comment:
Would it be possible to change this to a unique name? Maybe solveApolloniusTests. Turns out that having an array with the name tests in the tail causes all the tests to fail, as it calls the tests object instead.